ambry-store/src/test/java/com.github.ambry.store/IndexSegmentTest.java

```diff
@@ -245,17 +311,32 @@ private void addPutEntries(List<Long> offsets, long lastEntrySize, IndexSegment
    * @param numItems the expected numner of items the {@code indexSegment}
    * @param isMapped the expected mapped state of the {@code indexSegment}
    * @param endOffset the expected end offset of the {@code indexSegment}
+   * @param verifyLastModifiedTime {@code true} if last modified time needs to be verified for {@link PersistentIndex#VERSION_0},
+   *                                           {@code false} otherwise
+   * @param lastModifiedTimeInSecs the last modified time in secs
+   * @param resetKey the reset key for the index segment
    */
   private void verifyIndexSegmentDetails(IndexSegment indexSegment, Offset startOffset, int numItems, boolean isMapped,
-      long endOffset) {
+      long endOffset, boolean verifyLastModifiedTime, long lastModifiedTimeInSecs,
```

Comment:
why do you need the variable `verifyLastModifiedTime`? At any point the caller should be able to provide a valid `lastModifiedTimeInSecs` to check.